web/themes/custom/nysenate_theme/nysenate_theme.theme

```diff
@@ -1467,12 +1467,18 @@ function nysenate_theme_preprocess_views_view_unformatted(&$variables) {
         }
       }
 
+      $date = NULL;
+      if ($node->hasField('field_date') && !$node->get('field_date')->isEmpty()) {
```

Comment:
We've seen issues with using a field's `isEmpty()` method.  Change this to native detection and use null coalesce.